instagram/src/components/CommentSection/CommentForm.js

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+
+class CommentForm extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      newComment:"",
+      newID: ""
+    }
+  }
+
+  onClickButton = (event) => {
+    event.preventDefault();
+    console.log("onClickButton")
+    console.log(this.state.newComment)
+    this.props.addNewComment(this.state.newComment, this.state.newID)
+    // this.props.newID(this.state.newID)
+  }
+
+
+  onChangeComment = (event) => {
+    this.setState({
+      newComment: event.target.value,
```

Comment:
Right now, when you add a comment, it doesn't render an empty input field. This can be fixed by adding newComment: ""